Tensile/AsmUtils.py

```diff
@@ -181,19 +181,16 @@ def vectorStaticRemainder(rReg, dReg, divisor, tmpSgpr, comment=""):
         kStr += inst("_v_sub_u32", vgpr(rReg), vgpr(dReg), vgpr(rReg), comment)
     return kStr
 
-# only used for loop unroll and GlobalSplitU
+# only used for loop unroll and GlobalSplitU and XCC mapping
 # doRemainder==0 : compute quotient only
 # doRemainder==1 : compute quotient and remainder
 # doRemainder==2 : only compute remainder (not quotient unless required for remainder)
 # dreg == dividend
-# tmpSgpr must be 2 SPGRs
+# tmpSgpr must be 2 SPGRs (can be None if divisor is power of 2)
```

Comment:
Would you please change this comment (2 SPGRs -> 3)?
(SPGR -> SGPR....)

Reply:
Updated